### PR TITLE
Fix adding new location data into POI list

### PR DESCRIPTION
**Pull request by YarikMamykin**
Implements/Fixes #325

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
In case of appending POI list with new location data there was no update of POI list. 
The reason for such behavior - new location data appended into `SDL.NavigationModel.LocationDetails` array directly instead of using Ember object notification system. 
So now, location data appends through Ember object notification system.

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/controller/NavigationController.js [hunk 1]
```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
      * @param {Object} request
      */
     sendLocation: function(request) {
-      this.model.LocationDetails.push(
+      this.model.get('LocationDetails').addObject(
```
Comment (AKalinich-Luxoft) [inline app/controller/NavigationController.js]: @YarikMamykin try simply this
```suggestion
    this.model.LocationDetails.pushObject(
```

File: app/controller/NavigationController.js [hunk 2]
```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
      * @param {Object} request
      */
     sendLocation: function(request) {
-      this.model.LocationDetails.push(
+      this.model.get('LocationDetails').addObject(
         {
-          coordinate: {
+          coordinate: {
```
Comment (AKalinich-Luxoft) [inline app/controller/NavigationController.js]: @YarikMamykin please remove redundant spaces which github shows

**Review by AKalinich-Luxoft — CHANGES_REQUESTED** [review]

File: app/controller/NavigationController.js (hunk 1 above)
Comment (YarikMamykin) [reply app/controller/NavigationController.js]: @AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/277299046a476f95105d9c4ca0c3ca231377968c

File: app/controller/NavigationController.js (hunk 2 above)
Comment (YarikMamykin) [reply app/controller/NavigationController.js]: @AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/277299046a476f95105d9c4ca0c3ca231377968c

File: app/controller/NavigationController.js [hunk 3]
```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
      * @param {Object} request
      */
     sendLocation: function(request) {
-      this.model.LocationDetails.push(
+      this.model.LocationDetails.pushObject(
         {
           coordinate: {
             latitudeDegrees: request.params.latitudeDegrees,
-            longitudeDegrees: request.params.longitudeDegrees
+            longitudeDegrees: request.params.longitudeDegrees
```
Comment (AKalinich-Luxoft) [inline app/controller/NavigationController.js]: @YarikMamykin please also remove this space

File: app/controller/NavigationController.js [hunk 4]
```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
...
-            longitudeDegrees: request.params.longitudeDegrees
+            longitudeDegrees: request.params.longitudeDegrees 
           },
           locationName: request.params.locationName,
           addressLines: request.params.addressLines,
           locationDescription: request.params.locationDescription,
           phoneNumber: request.params.phoneNumber,
           locationImage: request.params.locationImage,
-          searchAddress: request.params.address
+          searchAddress: request.params.address
```
Comment (AKalinich-Luxoft) [inline app/controller/NavigationController.js]: @YarikMamykin and this one

Comment (YarikMamykin) [reply app/controller/NavigationController.js]: @AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/ccdf005934398ea67e5a62779fc851843b818e7c

File: app/controller/NavigationController.js (hunk 3 above)
Comment (YarikMamykin) [reply app/controller/NavigationController.js]: @AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/ccdf005934398ea67e5a62779fc851843b818e7c

**Review by AKalinich-Luxoft — APPROVED** [review]

**Review by mkorniichuk — APPROVED** [review]

**Comment by YarikMamykin** [comment]
Changes in PR approved. Will make PR to open sdl. https://github.com/smartdevicelink/sdl_hmi/pull/359

**Pull request closed without merging**